pom.xml

```diff
@@ -1980,7 +1980,7 @@
 
             <dependency>
                 <groupId>com.facebook.presto</groupId>
-                <artifactId>testing-mysql-server-5</artifactId>
+                <artifactId>testing-mysql-server-8</artifactId>
                 <version>${dep.testing-mysql-server-5.version}</version>
```

Comment:
In order to prevent needing to add the exclusion of the mysql client for every use of the `testing-mysql-server-8` dependency, can you just add the exclusion here?